Horizon/Horizon/Sources/Features/Dashboard/View/DashboardViewModel.swift

```diff
@@ -35,22 +35,22 @@ final class DashboardViewModel: ObservableObject {
     // MARK: - Init
 
     init(
-        programsInteractor: GetProgramsInteractor,
-        userInteractor: GetUserInteractor,
+        getCoursesInteractor: GetCoursesInteractor,
+        getUserInteractor: GetUserInteractor,
```

Comment:
Please change those properties to be `coursesInteractor` & `userInteractor`

Reply:
I'll leave them with get prefix as they better suit their purpose for now.